eng/pipelines/templates/steps/gpg-sign-and-flatten.yml

```diff
@@ -0,0 +1,35 @@
+parameters:
+  ArtifactDirectory: not-specified
+  OutputDirectory: not-specified
```

Comment:
Is OutputDirectory still a good parameter name.  It seems like OutputDirectory and FlattenedESRPDirectory are both output directories.  Both are gpg signed and one is flattened

Reply:
Honestly, it was the least amount of changes but yes, the OutputDirectory is still a good name. azure-sdk-for-android won't use or publish the FlattenedESRPDirectory, only the OutputDirectory, since that repository is still only single library publishing.